### ISSUE TITLE
Assessing single-cell transcriptomic variability through density-preserving data visualization

### ISSUE THREAD

**Issue by cmclean5** [cmclean5/PublicHealthModels#32]
## TL;DR

Nonlinear data visualization methods, such as t-distributed stochastic neighbor embedding (t-SNE) and uniform manifold approximation and projection (UMAP), summarize the complex transcriptomic landscape of single cells in two dimensions or three dimensions, but they neglect the local density of data points in the original space, often resulting in misleading visualizations where densely populated subsets of cells are given more visual space than warranted by their transcriptional diversity in the dataset. Here we present den-SNE and densMAP, which are density-preserving visualization tools based on t-SNE and UMAP, respectively, and demonstrate their ability to accurately incorporate information about transcriptomic variability into the visual interpretation of single-cell RNA sequencing data. Applied to recently published datasets, our methods reveal significant changes in transcriptomic variability in a range of biological processes, including heterogeneity in transcriptomic variability of immune cells in blood and tumor, human immune cell specialization and the developmental trajectory of Caenorhabditis elegans. Our methods are readily applicable to visualizing high-dimensional data in other scientific domains.

### Paper Link

https://www.nature.com/articles/s41587-020-00801-7#Abs1

### Author/Institution

Ashwin Narayan Department of Mathematics,  MIT)

## Cite

Narayan, A., Berger, B. & Cho, H. Assessing single-cell transcriptomic variability through density-preserving data visualization. Nat Biotechnol 39, 765–774 (2021). https://doi.org/10.1038/s41587-020-00801-7
